simple_history/manager.py

```diff
@@ -88,3 +89,19 @@ def _as_of_set(self, date):
                               history_type='-').exists():
                 continue
             yield last_change.instance
+
+    def bulk_history_create(self, objs, batch_size=None):
+        """Bulk create the history for the objects specified by objs"""
+
+        historical_instances = [
+            self.model(
+                history_date=getattr(instance, '_history_date', now()),
+                history_user=getattr(instance, '_history_user', None),
+                **{
+                    field.attname: getattr(instance, field.attname)
+                    for field in instance._meta.fields
```

Comment:
Is this where you need to account for `excluded_fields`?

Reply:
Yes. Going to get this merged and then ask @uadnan to rebase since we're still thinking about whether the solution there is best way to do things.